modules/tinymce/tools/docs/tinymce.js

```diff
@@ -14,6 +14,12 @@
  * @property {tinymce.AddOnManager} ThemeManager Global ThemeManager instance. Instance of AddOnManager.
  */
 
+/**
+ * Include EditorManager API docs.
+ *
+ * @include ../../src/core/main/ts/api/EditorManager.ts
```

Comment:
We should also comment out/remove the `borrows-member` part above as well, as we don't want that to be used if we're including it.

Reply:
Deleted in [`b995397` (#6775)](https://github.com/tinymce/tinymce/pull/6775/commits/b99539748bc82c00e104582adc37a9b0db558e01)